scanner/src/main/kotlin/scanners/scancode/ScanCode.kt

```diff
@@ -200,7 +200,8 @@ class ScanCode(
         }
 
         val result = getRawResult(resultsFile)
-        val summary = generateSummary(startTime, endTime, path, result)
+        val parseLicenseExpressions = scanCodeConfiguration["parseLicenseExpressions"]?.toBoolean() == true
```

Comment:
I personally find `?.toBoolean() ?: false` more readable.

Reply:
That actually reminds me to use our `isTrue()` extension function instead.